package.json

```diff
@@ -70,10 +69,10 @@
     "image-size": "0.7.4"
   },
   "devDependencies": {
-    "fork-ts-checker-webpack-plugin": "1.2.0",
+    "fork-ts-checker-webpack-plugin": "1.3.0",
```

Comment:
should hook up greenkeeper to keep these up to date

Reply:
https://aretecode.atlassian.net/jira/software/projects/OSWPN/boards/1?selectedIssue=OSWPN-56